lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -306,7 +300,52 @@ def test_vertical_coordinate_type(self):
 
 
 
-    ### Test compliance checks
+################################################################################
+#
+# IMOS 1.3 Checker
+#
+################################################################################
+
+class TestIMOS1_3(unittest.TestCase):
```

Comment:
Could this class inherit from TestUtils? Then you won't have to duplicate most of the code below.